SPECS/sshpass/sshpass.spec

```diff
@@ -1,13 +1,13 @@
-Summary:	Noninteractive ssh password provider
-Name:		sshpass
-Version:	1.09
-Release:       1%{?dist}
-License:	GPLv2+
-URL:		http://sourceforge.net/projects/sshpass/
-Source0:	http://downloads.sourceforge.net/project/sshpass/%{name}/%{version}/%{name}-%{version}.tar.gz
-Group:		Applications/Networking
+Summary:	    Noninteractive ssh password provider
```

Comment:
tabs vs spaces?  I think the alignment is off here.  Can you make sure this is all spaces?

Reply:
Yes, it was the tabs, thanks Jon